sorc/emcsfc_ice_blend.fd/emcsfc_ice_blend.f90

```diff
@@ -197,11 +195,11 @@ program emcsfc_ice_blend
  do j = 1, jmax
  do i = 1, imax
    if (dummy(i,j) < 0.1) then
-      mask_5min(i,j)=0   ! water
+      mask_5min(i,j)=0   !< water
```

Comment:
No, other than variable declarations, no doxygen is needed within the functions.

Reply:
removed.